Gemfile

```diff
@@ -47,3 +46,5 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
+
+gem 'bootstrap-sass'
```

Comment:
@naimaseales Install the following gems to help with debugging efforts in future:

```
  # In development, test group:
  gem 'pry-rails' # allows you to call binding.pry to "pry open" running code
  gem 'pry-inline' # automatically shows variable values inline in rails console
  gem 'rb-readline' # rb-readline makes pry history behave on all platforms (e.g. ensures ~/.pry_history auto loads)
```
```
  # In development group only
  gem 'better_errors' # Enhanced Rails error page with built-in rails console
  gem 'binding_of_caller' # Required by better_errors
  gem 'quiet_assets' # Less noisy but easier to debug server output
```